### ISSUE TITLE
Unable to resolve modules & Errors prevented startup

### ISSUE THREAD

**Issue by M165437** [meteor/todos#90]
I've just cloned the repo, ran `meteor` and got this error:

```
Unable to resolve some modules:

  "faker" in […]/todos/imports/api/todos/todos.js (web.browser)

Consider running: meteor npm install --save faker

=> Errors prevented startup:

   While minifying app stylesheet:
   error: Can't load npm module 'autoprefixer' from packages/minifier-postcss_plugin.js. Check your Npm.depends().

=> Your application has errors. Waiting for file change.
```

**Comment by Bandit** [comment]
I also get this error. Any ideas?

**Comment by benjamn** [comment]
You most likely need to run `npm install` (or `meteor npm install` if you don't have `npm` available) before starting the app.

**Comment by Bandit** [comment]
This is obvious in hindsight, however I tried to run `meteor create --example todos` and it told me to git clone this repo (so assumed there weren't any further install steps)

**Comment by tmeasday** [comment]
This would be more obvious once we implement https://github.com/meteor/meteor/issues/6322

**Comment by benjamn** [comment]
This should be fixed if you do `meteor update --release 1.3.1-rc.3`, soon to become 1.3.1.
